### PR TITLE
feat: :sparkles: consolidated app nav to settings & dropdown

### PR DESCRIPTION
**Pull request by joshistoast**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Community Node Submission

## Have you discussed this change with the InvokeAI team?
- [x] Yes
- [ ] No, because:

      
## Have you updated all relevant documentation?
- [ ] Yes
- [x] No

## Description

Moves all the application's top right nav elements into a new hamburger menu or into the settings modal.

## Related Tickets & Documents

- Related Issue #
- Closes #

## QA Instructions, Screenshots, Recordings

image
image

## Added/updated tests?

- [ ] Yes
- [x] No : I don't know how to do tests 👉👈

### CODE REVIEW

**Comment by joshistoast** [comment]
There's currently an issue where sometimes the localized language variable doesn't resolve properly and results in the original raw string:

image

**Comment by blessedcoolant** [comment]
> There's currently an issue where sometimes the localized language variable doesn't resolve properly and results in the original raw string:
> 
> image

Most likely an issue with hot reload at times. Refreshing the page should fix it.

**Comment by blessedcoolant** [comment]
Pushed a couple of fixes.

- Menu icons were minuscule. Increased the size.
- Scheduler list in settings for some reason was not showing the right labels. Fixed.

Everything else seems fine to me.

I'll let @psychedelicious have a quick look too before we merge.

Thanks for the PR.

**Review by blessedcoolant — APPROVED** [review]

File: invokeai/frontend/web/src/features/system/components/SiteHeader.tsx
Comment (psychedelicious) [inline invokeai/frontend/web/src/features/system/components/SiteHeader.tsx:50]: either no icons, or default icons size. the rest of the app's context menus use the default size, if you want to change them, please make that a follow up PR that changes them for the whole app. they need to be consistent

File: invokeai/frontend/web/src/features/system/components/SiteHeader.tsx
Comment (psychedelicious) [inline invokeai/frontend/web/src/features/system/components/SiteHeader.tsx:62]: `MenuItems` that are links need to be done like this to provide an accessible UI:
```ts
<Link href={imageDTO.image_url} target="_blank">
  <MenuItem icon={<FaExternalLinkAlt />}>
    {t('common.openInNewTab')}
  </MenuItem>
</Link>
```

File: invokeai/frontend/web/public/locales/en.json
Comment (psychedelicious) [inline invokeai/frontend/web/public/locales/en.json:1]: the only translation file that should ever be changed from the repo is `en.json`, the rest need to come from weblate; i'm impressed that you went and translated them all lol but only the english, pls remove the others

File: invokeai/frontend/web/src/features/system/components/SiteHeader.tsx
Comment (psychedelicious) [inline invokeai/frontend/web/src/features/system/components/SiteHeader.tsx:41]: don't think the menu button needs a tooltip

File: invokeai/frontend/web/src/features/system/components/SiteHeader.tsx
Comment (psychedelicious) [reply invokeai/frontend/web/src/features/system/components/SiteHeader.tsx:62]: also IMO all the links should open in a new tab, right now they open in the same tab

**Review by psychedelicious — CHANGES_REQUESTED** [review]
overall looks good just a few changes needed

File: invokeai/frontend/web/src/features/system/components/SiteHeader.tsx
Comment (blessedcoolant) [reply invokeai/frontend/web/src/features/system/components/SiteHeader.tsx]: Fixed.

File: invokeai/frontend/web/public/locales/en.json
Comment (blessedcoolant) [reply invokeai/frontend/web/public/locales/en.json:1]: Reverted.

File: invokeai/frontend/web/src/features/system/components/SiteHeader.tsx
Comment (blessedcoolant) [reply invokeai/frontend/web/src/features/system/components/SiteHeader.tsx:41]: Removed.

**Comment by maryhipp** [comment]
Can we please leave the ColorModeButton as an unused exported component for now?

**Comment by joshistoast** [comment]
> Can we please leave the ColorModeButton as an unused exported component for now?

Any reason to keep it around?

**Review by maryhipp — APPROVED** [review]
Found a workaround for colormode export - LGTM

**Review by psychedelicious — APPROVED** [review]

**Pull request merged**
